Subject: [PATCH] added verilated pci device to qemu Kconfig and meson.build
---
Index: hw/misc/Kconfig
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/hw/misc/Kconfig b/hw/misc/Kconfig
--- a/hw/misc/Kconfig	(revision 8333dba7326d89ce180221c4d0c6d34826a9a4a3)
+++ b/hw/misc/Kconfig	(date 1778530118961)
@@ -255,3 +255,8 @@
     bool
 
 source macio/Kconfig
+
+config VERILATED_PCI
+    bool
+    default y
+    depends on PCI
Index: hw/misc/meson.build
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/hw/misc/meson.build b/hw/misc/meson.build
--- a/hw/misc/meson.build	(revision 8333dba7326d89ce180221c4d0c6d34826a9a4a3)
+++ b/hw/misc/meson.build	(date 1778543910406)
@@ -167,3 +167,6 @@
 
 # HPPA devices
 system_ss.add(when: 'CONFIG_LASI', if_true: files('lasi.c'))
+
+# verilated pci
+system_ss.add(when: 'CONFIG_VERILATED_PCI', if_true: files('verilated-pci.c'))
